common/supplemental/units.xml

```diff
@@ -0,0 +1,247 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<!DOCTYPE supplementalData SYSTEM "../../common/dtd/ldmlSupplemental.dtd">
+<!--
+Copyright © 1991-2013 Unicode, Inc.
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+For terms of use, see http://www.unicode.org/copyright.html
+-->
+
+<supplementalData>
+	<version number="$Revision$"/>
+	<unitConstants>
+		<unitConstant constant="ft2m" value="0.3048"/>
+		<unitConstant constant="lb2kg" value="0.45359237"/>
+		<unitConstant constant="gravity" value="9.80665"/>
+		<unitConstant constant="PI" value="3.1415926535897932384626433832795" status='approximate'/>
+		<unitConstant constant="cup2m3" value="231*ft2m*ft2m*ft2m/16*12*12*12"/>
+		<unitConstant constant="G" value="6.67408E-11" status='approximate'/>   <!-- https://arxiv.org/pdf/1510.07674.pdf see 2014 CODATA -->
+	</unitConstants>
+	<unitQuantities> 
+		<!-- NB: quantity ordering is used in canonical order of derived units. -->
+		<!--  SI Base Units -->
+        <unitQuantity baseUnit='candela' quantity='luminous-intensity' status='simple'/>
```

Comment:
There is a mix of spaces and tabs for indenting here, please fix

Reply:
done